packages/sdk/.gitignore

```diff
@@ -2,4 +2,4 @@
 dist/
 tsconfig.tsbuildinfo
 .nyc_output/
-coverage/
+.coverage/
```

Comment:
this should be in the global get ignore as a wildcard. not replicated in each package